<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debugger</class>
 <widget class="QMainWindow" name="Debugger">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>425</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>425</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Debugger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>2</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>ï£¿+R</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepButton">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>2</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>S</string>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="frameButton">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>2</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>F</string>
    </property>
    <property name="text">
     <string>Run One Frame</string>
    </property>
    <property name="shortcut">
     <string>F</string>
    </property>
   </widget>
   <widget class="QLabel" name="registersLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Registers</string>
    </property>
   </widget>
   <widget class="QLabel" name="disassemblerLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Disassembler</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="registersTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>146</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>6</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>21</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>PC</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>AF</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>BC</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DE</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>HL</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>SP</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="disassemblerTable">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>371</width>
      <height>146</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>122</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>01</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>02</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>03</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>04</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>05</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Instruction</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="videoRegistersTable">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>121</width>
      <height>346</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>LCDC</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>STAT</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>SCY</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>SCX</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>LY</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>LYC</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DMA</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>BGP</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>OBP0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>OBP1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>WY</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>WX</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>BCPS</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>BCPD</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>OCPS</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>OCPD</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Addr</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>FF40</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>FF41</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>FF42</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>FF43</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>FF44</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>FF45</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>FF46</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>FF47</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>FF48</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>FF49</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>FF4A</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>FF4B</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>FF68</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>FF69</string>
     </property>
    </item>
    <item row="14" column="0">
     <property name="text">
      <string>FF6A</string>
     </property>
    </item>
    <item row="15" column="0">
     <property name="text">
      <string>FF6B</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="otherRegistersTable">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>131</width>
      <height>366</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>P1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>SB</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>SC</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DIV</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>TIMA</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>TMA</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>TAC</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>KEY1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>VBK</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>HDMA1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>HDMA2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>HDMA3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>HDMA4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>HDMA5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>SVBK</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>IF</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>IE</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Addr</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>FF00</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>FF01</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>FF02</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>FF04</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>FF05</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>FF06</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>FF07</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>FF4D</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>FF4F</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>FF51</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>FF52</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>FF53</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>FF54</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>FF55</string>
     </property>
    </item>
    <item row="14" column="0">
     <property name="text">
      <string>FF70</string>
     </property>
    </item>
    <item row="15" column="0">
     <property name="text">
      <string>FF0F</string>
     </property>
    </item>
    <item row="16" column="0">
     <property name="text">
      <string>FFFF</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="videoRegistersLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Video Registers</string>
    </property>
   </widget>
   <widget class="QLabel" name="otherRegistersLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Other Registers</string>
    </property>
   </widget>
   <widget class="QLabel" name="memoryLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>205</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="memoryTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>451</width>
      <height>186</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>17</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>17</number>
    </attribute>
    <row>
     <property name="text">
      <string>00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>01</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>02</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>03</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>04</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>05</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>06</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>07</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>08</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Addr</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>xA</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>xB</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>xC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>xD</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>xE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>xF</string>
     </property>
    </column>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="memoryEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>203</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>NNNN</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="breakpointButton">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>2</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Breakpoints</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addBrkpButton">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>365</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delBrkpButton">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>390</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="brkpEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>345</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="brkpList">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>81</width>
      <height>291</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="secondButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>2</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Shift+S</string>
    </property>
    <property name="text">
     <string>Run One Second</string>
    </property>
    <property name="shortcut">
     <string>Shift+S</string>
    </property>
   </widget>
   <widget class="QLabel" name="breakpointsLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Breakpoints</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
